js/views/device_pairing_dialog_view.js

```diff
@@ -67,10 +83,27 @@
       this.pubKey = this.pubKeyRequests.pop();
     },
     showView() {
+      const defaultView = this.$('.defaultView');
       const waitingForRequestView = this.$('.waitingForRequestView');
       const requestReceivedView = this.$('.requestReceivedView');
       const requestAcceptedView = this.$('.requestAcceptedView');
-      if (this.accepted) {
+      if (!this.isListening) {
+        const ourPubKey = textsecure.storage.user.getNumber();
+        defaultView.show();
+        requestReceivedView.hide();
+        waitingForRequestView.hide();
+        requestAcceptedView.hide();
+        // eslint-disable-next-line more/no-then
+        libloki.storage.getSecondaryDevicesFor(ourPubKey).then(pubKeys => {
```

Comment:
Would doing `async/await` on this function work instead of `.then`?

Reply:
Right, initially I didn't want to use await because I thought I couldn't make the function async, but I absolutely can in fact :)